inc/namespace.php

```diff
@@ -115,12 +115,11 @@ function check_consent() {
  *
```

Comment:
@jazzsequence There needs to be a doc comment for the `$headers` parameter ([as per tests](https://github.com/pantheon-systems/pantheon-edge-integrations-consent-management/runs/5790864551?check_suite_focus=true#step:4:16)).